stingray/pulse/fftfit.py

```diff
@@ -0,0 +1,144 @@
+import numpy as np
+from scipy.optimize import minimize, brentq
+
+
+def find_delay_with_ccf(amp, pha):
+    nh = 32
+    nprof = nh * 2
+    CCF = np.zeros(64, dtype=np.complex)
+    CCF[:nh] = amp[:nh] * np.cos(pha[:nh]) + 1.0j * amp[:nh] * np.sin(pha[:nh])
+    CCF[nprof: nprof - nh: -1] = np.conj(CCF[nprof: nprof - nh: -1])
+    CCF[nh // 2: nh] = 0
+    CCF[nprof - nh // 2: nprof - nh: -1] = 0
+    ccf = np.fft.ifft(CCF)
+
+    imax = np.argmax(ccf.real)
+    shift = normalize_phase_0d5(imax / nprof)
+
+    return shift
+
+
+def best_phase_func(tau, amp, pha, ngood=20):
+    # tau = params['tau']
+    # good = slice(1, idx.size // 2 + 1)
```

Comment:
Are these commented-out lines left-overs? Should they be deleted?

Reply:
Done